docker/dirigible/versions.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+cloudbuild:
+  enable_parallel: false
+versions:
+- dir: 6/debian11/6.3
+  from: marketplace.gcr.io/google/debian11
+  packages:
+    dirigible:
+      version: 6.3.18
```

Comment:
I see that from the time you've pushed this PR until today we have a newer version. 6.3.26. Can you please update it? Thanks!

Reply:
Updated!